rules/prefer-await.js

```diff
@@ -0,0 +1,210 @@
+'use strict';
+const {methodCallSelector} = require('./selectors/index.js');
+const {removeParentheses, removeMethodCall} = require('./fix/index.js');
+const {getParenthesizedText, isParenthesized} = require('./utils/parentheses.js');
+const shouldAddParenthesesToCallExpressionCallee = require('./utils/should-add-parentheses-to-call-expression-callee.js');
+const getFunctionParameterVariables = require('./utils/get-function-parameter-variables.js');
+const needsSemicolon = require('./utils/needs-semicolon.js');
+
+const MESSAGE_ID_ERROR = 'prefer-await/error';
+const MESSAGE_ID_SUGGESTION = 'prefer-await/suggestion';
+const messages = {
+	[MESSAGE_ID_ERROR]: 'Do not use `Promise#{{method}}(…)`.',
+	[MESSAGE_ID_SUGGESTION]: 'Use `await` expression (UNSAFE).',
```

Comment:
```suggestion
	[MESSAGE_ID_SUGGESTION]: 'Use `await` operator (UNSAFE FIX).',
```

To make it clear that it's the "fix" that is unsafe, not using "await".